certbot/plugins/manual.py

```diff
@@ -44,18 +77,26 @@ class Authenticator(common.Plugin):
 
 Before continuing, verify the record is deployed."""
     _HTTP_INSTRUCTIONS = """\
-Create a file containing just this data:
+Make sure your web server displays the following content at
+{uri} before continuing:
 
 {validation}
-
-And make it available on your web server at this URL:
-
-{uri}
```

Comment:
I think there was a bad merge here as this reverts some of the changes made in #4810.